suite/tests/client-interface/int64_overrides.template

```diff
@@ -0,0 +1,2 @@
+1
+Hello, world!
```

Comment:
This should just a `.expect` file, right?  https://github.com/DynamoRIO/dynamorio/wiki/Test-Suite#test-output

Reply:
Done